source/server/listener_manager_impl.cc

```diff
@@ -553,9 +561,23 @@ void ListenerManagerImpl::startWorkers(GuardDog& guard_dog) {
   }
 }
 
-void ListenerManagerImpl::stopListeners() {
-  for (const auto& worker : workers_) {
-    worker->stopListeners();
+void ListenerManagerImpl::stopListeners(StopListenersType listeners_type) {
```

Comment:
nit: s/listeners_type/stop_listeners_type (this will fall clang-tidy at some point)

Reply:
sorry. missed this. Will change